catalog/app/containers/Bucket/Meta.tsx

```diff
@@ -2,12 +2,27 @@ import cx from 'classnames'
 import * as R from 'ramda'
 import * as React from 'react'
 import * as M from '@material-ui/core'
+import * as Lab from '@material-ui/lab'
 
 import JsonDisplay from 'components/JsonDisplay'
 import AsyncResult from 'utils/AsyncResult'
+import { Json } from 'utils/types'
 
 import Section, { SectionProps } from './Section'
 
+interface MetaData {
+  message: string
+  user_meta?: object
```

Comment:
```suggestion
  user_meta?: JsonRecord
```
(`JsonRecord` can be imported from `utils/types`)